examples/multimodal_agent.py

```diff
@@ -44,12 +44,30 @@ async def get_weather(
                     return f"The weather in {location} is {weather_data}."
                 else:
                     raise Exception(
-                        "Failed to get weather data, status code: {response.status}"
+                        f"Failed to get weather data, status code: {response.status}"
                     )
 
     await ctx.connect(auto_subscribe=AutoSubscribe.AUDIO_ONLY)
+    participant = await ctx.wait_for_participant()
 
-    assistant = multimodal.MultimodalAgent(
+    # to use Microsoft Azure, uncomment the following lines
+    # agent = multimodal.MultimodalAgent(
+    #     model=openai.realtime.RealtimeModel.with_azure(
+    #         azure_deployment="<model-deployment>",
+    #         azure_endpoint="wss://<endpoint>.openai.azure.com/", # or AZURE_OPENAI_ENDPOINT
+    #         api_key="<api-key>", # or AZURE_OPENAI_API_KEY
+    #         api_version="2024-10-01-preview", # or OPENAI_API_VERSION
```

Comment:
I think this is azure specific

Reply:
Yeah this is weird but this is what the official OpenAI sdk is using

Reply:
Keeping the same env vars since this is what is being documented on their docs